### ISSUE TITLE
[BUG] HotReload command stacking

### ISSUE THREAD

**Issue by K4ryuu** [roflmuffin/CounterStrikeSharp#66]
### Description

Whenever you drag and drop the new files onto your server from the same project, the server reloads the plugin, right? Well yes, but actually the earlier data stay in the server memory. That means when you run a command, first get gets executed once, then twice and so on as many times as you upload a new build without restart or so.

### Reproduce

1. Create a simple command
2. Upload it to the server
3. Execute it (it will print once)
4. Upload the files again (estimated modification on plugin or so)
5. When you execute the command it is executed twice
6. ...

### EDIT

In addition, because probably it is connected to this bug, when you upload the new files or run `css_plugins reload xy`, it says the following:

```
css_plugins reload PluginName
Could not reload plugin "PluginName")
```

**Comment by KillStr3aK** [comment]
maybe related to fe236806e1ca35f8efcbd4697aa7fdea70a3c80c?

**Comment by KillStr3aK** [comment]
About reloading and unloading, you can only use these commands with the plugin id and not by name.

**Comment by K4ryuu** [comment]
Auto reload when dragging new dll in. Not by command

**Comment by Cruze03** [comment]
Yeah, I've noticed the same in my test plugin

**Comment by roflmuffin** [comment]
Is it related to #53 ?

**Comment by K4ryuu** [comment]
Well it is, but additionally it also cannot unload the earlier command if you use the command helper or required permission as I tested so far. You don't have to use multiple commands, but any multiple property instead.

So basically this also does the magic:
```
[ConsoleCommand("rank", "Check the current rank and points")]
[CommandHelper(whoCanExecute: CommandUsage.CLIENT_ONLY)]
```

**Comment by K4ryuu** [comment]
I'm not sure, but I will test if the problem is only with multiple commands or for any other property. Maybe I'm wrong about this, but as I remember it also appears with one of my command, which have only one command and one command helper property.

I am going to look after that.

**Comment by lukepolo** [comment]
ive noticed it in odd situations 

example


```
[ConsoleCommand("css_r", "Toggles the player as ready")]
    [ConsoleCommand("css_ready", "Toggles the player as ready")]
    [CommandHelper(whoCanExecute: CommandUsage.CLIENT_ONLY)]
    public void OnReady(CCSPlayerController? player, CommandInfo? command)
    {
        if (!IsWarmup() || player == null)
        {
            return;
        }

        if (!_readyPlayers.ContainsKey(player.UserId!.Value))
        {
            _readyPlayers[player.UserId.Value] = true;
        }
        else
        {
            _readyPlayers[player.UserId.Value] = !_readyPlayers[player.UserId.Value];
        }

        if (TotalReady() == 10)
        {
            UpdatePhase(ePhase.Knife);
        }

        SendReadyMessage(player);

        SendNotReadyMessage();
    }
    ```


typingin !r will go nuts and spit out lots of repeated messages  (after a couple of hot reloads)

**Comment by K4ryuu** [comment]
With V65 the problem still appears. 😒

**Comment by roflmuffin** [comment]
Fixed by #321 as we no longer key by the callback
